src/main/java/com/kalessil/phpStorm/phpInspectionsEA/inspectors/apiUsage/arrays/LowPerformanceArrayUniqueUsageInspector.java

```diff
@@ -12,8 +12,8 @@
 import org.jetbrains.annotations.NotNull;
 
 public class LowPerformanceArrayUniqueUsageInspector extends BasePhpInspection {
-    private static final String strProblemUseArrayKeysWithCountValues = "array_keys(array_count_values(<expression>)) will be more performing (but provide comments)";
-    private static final String strProblemUseCountWithCountValues     = "count(array_count_values(<expression>)) will be more performing (but provide comments)";
+    private static final String strProblemUseArrayKeysWithCountValues = "array_keys(array_count_values(...)) would be more efficient.";
```

Comment:
I'm not entirely sure it should be the responsibility of PHPInspectionEA to tell developers to add comments when they do these types of changes, which is the reason why I removed that part of the message. I think it should be left at the developer's discretion whether or not a change should receive a comment. I don't mind adding it back, however I'd definitely like to understand your rationale as to why in this case and not any of the other inspections.